yarn.lock

```diff
@@ -310,19 +310,14 @@
   dependencies:
     eslint-visitor-keys "^3.3.0"
 
-"@eslint-community/regexpp@^4.4.0":
-  version "4.4.0"
-  resolved "https://registry.npmjs.org/@eslint-community/regexpp/-/regexpp-4.4.0.tgz"
```

Comment:
@wboereboom any particular reason you updated this yarn file in your pr #1250 ?

Reply:
I ran npm install to run unit tests and I comitted the file with the PR.
Seems like it hadn't been updated for a while.